keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -1462,225 +1278,309 @@ prevented. Deleting the entire cluster also leaves resources allocated outside
 of the cluster in an allocated state.
 
 ```go
-// ResourceClaimSpec defines how a resource is to be allocated.
 type ResourceClaimSpec struct {
-    // ResourceClassName references the driver and additional parameters
-    // via the name of a ResourceClass that was created as part of the
-    // driver deployment.
+    // This field holds configuration for multiple potential drivers which
+    // could satisfy requests in this claim. The configuration applies to
+    // the entire claim. It is ignored while allocating the claim.
+    //
     // +optional
-    ResourceClassName string
+    // +listType=atomic
+    Config []ConfigurationParameters
```

Comment:
I would list this last for aesthetic reasons

Reply:
Okay.